src/test/skript/tests/syntaxes/effects/EffPotion.sk

```diff
@@ -0,0 +1,15 @@
+test "potion effect":
+	spawn a pig at spawn of world "world"
+	set {_pig} to last spawned pig
+	set {_potion} to potion effect of slowness of tier 5 for 10 seconds
+	apply {_potion} to player
+	assert {_pig} has potion effect slowness with "pig failed to apply slowness"
+	apply potion effect of speed of tier 5 for 10 seconds to player
+	assert {_pig} has potion effect speed with "pig failed to apply speed"
+	apply haste of tier 5 to player for 10 seconds
```

Comment:
There is no player in a test.

Reply:
Appears more than once

Reply:
oops, guess I didn't realize that one I'll fix it up rn